manifests/p/ProjectJupyter/JupyterLab-Desktop/3.5.0-1/ProjectJupyter.JupyterLab-Desktop.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ProjectJupyter.JupyterLab-Desktop
+PackageVersion: 3.5.0-1
+PackageLocale: en-US
+Publisher: Project Jupyter
+PackageName: JupyterLab-Desktop

```

```suggestion
PackageName: JupyterLab
```
Must match Control Panel / Apps&Features